tests/regrtest_data/long_test_file.py

```diff
@@ -0,0 +1,100 @@
+
+
+
+
+
+
```

Comment:
```suggestion
"""
This file is used for bad-option-value's test. We needed a module that isn’t restricted by line numbers
as the enable tests are using various line numbers to test the behaviour.

Using an empty module creates issue as you can’t disable something on line 10 if it doesn’t exist. Thus, we created an extra long file so we never run into issue with that.
"""
```

Reply:
I'm going to commit this and see if tests pass. We use those first lines quite a bit in the test, so I'm not sure if those can be a doctoring.